### ISSUE TITLE
 Failed to release to TestPyPI

### ISSUE THREAD

**Issue by bugen** [bugen/pypipe#33]
https://github.com/bugen/pypipe/actions/runs/7209518193

**Comment by bugen** [comment]
@diazona
The error this time seems to be on the TestPyPI side.  I created release v0.4.1a0 on GitHub without making any changes, and it was successfully released to both TestPyPI and PyPI.

In cases like this, where a release fails for some reason, I would like to manually rerun the release.yml without creating a new release on GitHub. Is this possible?"

**Comment by diazona** [comment]
Certainly. I think you can just [rerun the job](https://docs.github.com/en/actions/managing-workflow-runs/re-running-workflows-and-jobs#re-running-failed-jobs-in-a-workflow) using the Github web interface (or the REST API). In that case it will re-attempt the deployment to Test PyPI, then if that succeeds, it will go ahead to the next step and deploy to PyPI.

If you have any trouble with those instructions, let me know, and I can take a closer look and maybe offer some other suggestions.

**Comment by bugen** [comment]
I didn't want to leave the situation where I couldn't release on PyPI unattended, so I hurriedly dealt with it. As a result, a meaningless version v0.4.1 was created...

> If you have any trouble with those instructions, let me know, and I can take a closer look and maybe offer some other suggestions. 
Thank you👍 It's a bad habit of mine to try to handle things in my way without fully understanding. From now on, I will make it a point to consult calmly before taking any action.

**Comment by diazona** [comment]
Makes sense. In general there is rarely a reason to do anything in a rush when it comes to open source software, or at least I think so. But no harm done in this case; it doesn't hurt to bump the version number for no reason.
